op-node/rollup/types.go

```diff
@@ -88,6 +88,10 @@ type Config struct {
 	// Active if EcotoneTime != nil && L2 block timestamp >= *EcotoneTime, inactive otherwise.
 	EcotoneTime *uint64 `json:"ecotone_time,omitempty"`
 
+	// KromaMPTTime sets the activation time of the Kroma MPT network upgrade.
+	// Active if KromaMPTTime != nil && L2 block timestamp >= *KromaMPTTime, inactive otherwise.
+	KromaMPTTime *uint64 `json:"kroma_mpt_time,omitempty"`
```

Comment:
```suggestion
	KromaMptTime *uint64 `json:"mpt_time,omitempty"`
```

Reply:
I thought `kroma_mpt_time` is better since it's easy to recognize that it's kroma's own config. wdyt?